components/channel/add-member.tsx

```
@@ -0,0 +1,34 @@
+  return (
+    <AddModal
+      action={() => { }}
+      actionLabel="Add"
+      title="Add Member"
+      content={
+        <>
+          <TextInput
+            label="User id"
+            mode="flat"
+          />
+        </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)